pkg/network/setup/netconf_test.go

```diff
@@ -121,9 +128,35 @@ var _ = Describe("netconf", func() {
 			Name:          testNetworkName,
 			NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}},
 		}}
+		vmi.Status.Interfaces = []v1.VirtualMachineInstanceNetworkInterface{
+			{Name: testNetworkName, PodInterfaceName: primaryPodIfaceName},
+		}
 		Expect(netConf.Setup(vmi, vmi.Spec.Networks, launcherPid, netPreSetupDummyNoop)).NotTo(Succeed())
 	})
 
+	DescribeTable("fails the setup run when interface status is missing", func(interfaceStatues []v1.VirtualMachineInstanceNetworkInterface) {
```

Comment:
I do not understand why it should fail on this.
As I see it, it should ignore such network and not run setup on them.

When the information is available, then it should do it.

Reply:
You can resolve this.

We should fail and not ignore because we do not what this to reach the virt-launcher SyncVMI.